getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -13,38 +13,6 @@ create content. It uses a syntax similar to
 to be optimized for and tightly integrated with Godot Engine, allowing great
 flexibility for content creation and integration.
 
-History
```

Comment:
I would add a `.. seealso::` that references the new page. While I agree that it's not *necessary* to know GDScript history to learn the basics, this page is the one most people will start with when faced with the question "What is GDScript" and the subsequent question many have: "Why the hell does Godot have a DSL instead of using <insert favourite language>?". So it's important that this information is easy to find.